keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -197,19 +199,9 @@ spec:
   protocol: [10]
     name:
     version:
-    azureBlob:
-      containerName:
-      storageAccount:
-    s3:
-      endpoint:
-      bucketName:
-      region:
-      signatureVersion:
-    gs:
-      bucketName:
-      privateKeyName:
-      projectId:
-      serviceAccount:
+    bucketName:
+    endpoint:
+    parameters:
```

Comment:
Is the idea that object-store-specific fields (eg. signatureVersion) will now exist in `parameters`?

Reply:
Same question here.

Reply:
Yes.  We are getting rid of any protocol specific fields and moving them to parameters.  We can't hope to keep up to date with protocol changes/needs so we are keeping a common set of named params and moving everything else to a map[string]string.  This allows protocols to change w/o the api needing to change.

Reply:
I think that makes sense.